geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js

```diff
@@ -78,8 +78,8 @@ class Canvas extends Component {
     }
 
     if (prevProps !== this.props) {
```

Comment:
@ddelpiano why test all props if you only take some? I think checking only the props that you will set in this `if` should be tested here.

e.g. if a Canvas handler prop (e.g. `cameraHandler`) is changed then we should not update the threeDEngine, or am I missing something?

Reply:
[technical debt](https://github.com/MetaCell/geppetto-meta/issues/205) for the future, not a priority right now.